ophidian/src/ophidian/finder.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+import os
+import re
+import sys
+import glob
+import json
+import subprocess
+
+import attr
+
+
+VALID_PYTHON_NAMES = re.compile(r"""
+    ^
+    (python|pypy)  # We currently only support these two implementations
+    (-?(\d\.?)+)?
+    (\.exe)?
```

Comment:
Since many of the path-component things in the tools (especially the bash) is Unix-only, is this here for forward-looking reasons only?